packages/components/rollup.config.js

```diff
@@ -55,8 +55,8 @@ export default {
     }),
     copy({
       targets: [
-        { src: 'LICENSE', dest: 'dist' },
-        { src: 'README.md', dest: 'dist' },
+        { src: '../../LICENSE', dest: 'dist' },
+        { src: '../../README.md', dest: 'dist' },
```

Comment:
Don't we want the README to be separate for both the hooks and the components package? 🤔

Reply:
Maybe someday? but we only have one README file today :P So might as well use that shared one everywhere for now.

And this will probably all change with our new build/publishing system plans, this was mainly to keep the status quo from before the monorepo work